index.html

```diff
@@ -17,29 +17,175 @@
+        </article>
+        <article class="dark-bg">
+          <img src="images/Rectangle21(4).png" alt="project image" />
+          <div class="project project5">
+            <h1 class="heading2">Project name goes here</h1>
+            <div class="lang-container">
+              <p class="languages">HTML/CSS</p>
+              <p class="languages">Ruby on Rails</p>
+              <p class="languages">JavaScript</p>
+            </div>
+            <button class="buttons-primary-white" type="button">
+              See this project &#160;
+              <i class="fa fa-arrow-right"></i>
+            </button>
+          </div>
+        </article>
+        <article class="dark-bg">
+          <img src="images/Rectangle21(5).png" alt="project image" /> 
+          <div class="project project6">
+            <h1 class="heading2">Project name goes here</h1>
+            <div class="lang-container">
+              <p class="languages">HTML/CSS</p>
+              <p class="languages">Ruby on Rails</p>
+              <p class="languages">JavaScript</p>
+            </div>
+            <button class="buttons-primary-white" type="button">
+              See this project &#160;
+              <i class="fa fa-arrow-right"></i>
+            </button>
+          </div>
+        </article>
+      </section>
+      <section class="white-bg">
+        <h1 class="heading">About</h1>
+        <p class="para">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+        <button class="buttons-primary-dark" type="button">
+          Start collaboration &#160; <i class="fa fa-download" aria-hidden="true"></i>
+        </button>
+        <div class="about-container">
+        <article class="blue about">
+          <h1 class="heading3">Languages</h1>
+          <ul class="para2">
+            <li>JavaScript (ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+          <img class="icon-bg" src="images/ic_languages_mob.svg" alt="#">
+        </article>
+        <article class="red about">
+          <h1 class="heading3">Frameworks</h1>
+          <ul class="para2">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+          <img class="icon-bg" src="images/ic_framework.svg" alt="#">
+        </article>
+        <article class="yellow about">
+          <h1 class="heading3">Skills</h1>
+          <ul class="para2">
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>Github</li>
+            <li>Gitlab</li>
+            <li>TErminal</li>
+          </ul>
+          <img class="icon-bg" src="images/ic_skills.svg" alt="#">
+        </article>
+      </div>
```

Comment:
- [ ] It would be better to use `ul` and `li` for categories in `About myself` section to improve structure and semantics. Languages, Frameworks, and Skills should be list items. kindly change `article` to `li`.

Reply:
Hi @medaminedev66 thank you for your feedback.
regarding this point, I can not change the article to ul and li as it has more than a list inside. alternatively, I can change it to div and h2 and ul to fit the content.
Kindly, inform if that would work or how to implement your opinion.

Reply:
Hi @Elerqsousy, I understand your point. You can change `<div class="about-container">` to `<ul class="about-container">` and `<article class="blue about">` to `<li class="blue about">`.
- The `li` has default styles you may not need them. What you need to do is to disable them. try to implement the next lines:
```css
li {
  list-style: none;
}